ui/apps/platform/src/Containers/Policies/Wizard/Step5/ReviewPolicyForm.tsx

```diff
@@ -138,7 +138,7 @@ function ReviewPolicyForm({
                 </Flex>
                 <Divider component="div" />
                 <FlexItem className="pf-v5-u-p-lg">
-                    <PolicyDetailContent policy={postFormatExclusionField(values)} isReview />
```

Comment:
The `PolicyDetailContent` component only needs fields that are common to both `Policy` and `ClientPolicy`, so its prop type was updated to reflect this and the unnecessary conversion here was removed.